src/UserModE131.h

```diff
@@ -0,0 +1,67 @@
+#include <ESPAsyncE131.h>
+
+
+
+class UserModE131:public Module {
+
+public:
+
+  UserModE131() :Module("e131/sACN support") {
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  };
+
+  //setup filesystem
+  void setup() {
+    Module::setup();
+    print->print("%s %s\n", __PRETTY_FUNCTION__, name);
+
+    if(!net->interfacesInited) {
+      print->print("No network yet, can't start e131");
+    }
+
+//    print->print("%s %s %s\n", __PRETTY_FUNCTION__, name, success?"success":"failed");
+  }
+
+  void loop(){
+    // Module::loop();
+    if(!net->interfacesInited) {
+      return;
+    }
+    if(!e131Created) {
+      print->print("Network exists, now init e131\n");
+      e131 = ESPAsyncE131(universeCount);
```

Comment:
Calling the constructor before we have network causes the library to freak out, so need to delay. This is a but ugly to do here in loop though

Reply:
See above